languages.js

```diff
@@ -262,233 +284,264 @@ const translations = {
-    "userText": "Klikněte sem pro úpravu",
-    // End of Body and New Tab Items
-},
-        
+        // Menu Items
+        "feedback": "Zpětná vazba",
+        "resetsettings": "Resetovat nastavení",
+        menuCloseText: 'Zavřít',
+        // Shortcuts
+        "shortcutsText": "Zkratky",
+        "enableShortcutsText": "Zapne/vypne zkratky",
+        "editShortcutsText": "Upravit zkratky",
+        "editShortcutsList": "Upravit zkratky",
+        "shortcutsInfoText": "Vyberte, které zkratky se mají zobrazit",
+        "adaptiveIconText": "Adaptivní tvary ikon",
+        "adaptiveIconInfoText": "Ikony zkratek budou vždy kulaté",
+        "ai_tools_button": "AI nástroje",
+        "enable_ai_tools": "Zapne/vypne zkratky AI nástrojů",
+        // Digital Clock
+        "digitalclocktittle": "Digitální hodiny",
+        "digitalclockinfo": "Zapne/vypne digitální hodiny",
+        "timeformattittle": "Použít 12hodinový formát",
+        "timeformatinfo": "Použije se 12hodinový formát času",
+        // Misc
+        "fahrenheitCelsiusCheckbox": "Přepnout na stupně Fahrenheita",
+        "fahrenheitCelsiusText": "Změny se projeví po obnovení stránky",
+        "search_suggestions_button": "Návrhy ve vyhledávání",
+        "search_suggestions_text": "Zapne/vypne návrhy vyhledávání",
+        // Proxy
+        "useproxytitletext": "Obcházení proxy",
+        "useproxyText": "Pokud nefungují návrhy ve vyhledávání",
+        "ProxyText": "Proxy pro obcházení CORS",
+        "ProxySubtext": "Nastavte si vlastní proxy pro obcházení CORS",
+        "HostproxyButton": "Provozování vlastní proxy",
+        "saveproxy": "Uložit",
+        // Location
+        "UserLocText": "Zadejte svou polohu",
+        "UserLocSubtext": "Pokud není správná poloha počasí",
+        "userLoc": "Vaše poloha (město / zeměpisná šířka, zeměpisná délka)",
+        "InputOptionsButton": "Co lze zadat",
+        "saveLoc": "Uložit",
+        // Weather
+        "WeatherApiText": "Zadejte svůj klíč k WeatherAPI",
+        "WeatherApiSubtext": "Pokud nefunguje funkce počasí",
+        "userAPI": "Váš klíč k WeatherAPI",
+        "LearnMoreButton": "Zjistit více",
+        "saveAPI": "Uložit",
+        // End of Menu Items
+
+        // Body Items
+        // Calendar
+        "days": ['neděle', 'pondělí', 'úterý', 'středa', 'čtvrtek', 'pátek', 'sobota'],
+        "months": ['Leden', 'Únor', 'Březen', 'Duben', 'Květen', 'Červen', 'Červenec', 'Srpen', 'Září', 'Říjen', 'Listopad', 'Prosinec'],
+        // End of Calendar
+
+        // Weather
+        "humidityLevel": "Vlhkost",
+        "feelsLike": "Pocitová teplota",
+        "location": "Země",
+        // End of Weather
+
+        // New Tab Item
+        "conditionText": "Dobrý den! Jak se máte?",
+        "enterBtn": "Vyhledat",
+        "searchPlaceholder": "Zadejte hledaný výraz...",
+        "searchWithHint": "Vyhledávat prostřednictvím",
+        "ai_tools": "AI nástroje",
+        "userText": "Upravíte kliknutím",
```

Comment:
```suggestion
        "userText": "Upravíte po kliknutí",
```
Hah, this is even better :D.